src/WebJobs.Script.WebHost/Controllers/ExtensionsController.cs

```diff
@@ -58,6 +62,11 @@ public async Task<IActionResult> Get()
         [Route("admin/host/extensions/{id}")]
         public async Task<IActionResult> Delete(string id)
         {
+            if (_extensionBundleManager.IsExtensionBundleConfigured())
+            {
```

Comment:
As discussed in person, please add tests

Reply:
There aren't any tests cases for any of the controller yet. We might need to setup something in e2e tests. Have added test cases for changes to extension manager in the extension manager tests.